pyproject.toml

```diff
@@ -27,6 +27,7 @@ sqlalchemy = { version = ">=2.0.21", optional = true }
 openpyxl = "^3.0.10"
 alembic = { version = "^1.0.11", optional = true }
 urllib3 = ">=1.26"
+certifi = "2024.7.4"
```

Comment:
Okay, so I checked where this dependency come from. It is a transitive dependency of `requests` package. Also, I checked the `requests` itself - it's latest version also uses old `certifi`. So I think in this case it's okay to add a consraint to our project and wait for `requests` upgrade. Two more things to do:

1. add a comment that once `requests` updates `certifi` to the version that will satisfy us - we should roll this change back and upgrae `requests instead
2. also, instead of pinning the exact version - better relax the constraint a bit (e.g. "^2024.7.4")